ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -290,6 +297,8 @@ public StoreConfig(VerifiableProperties verifiableProperties) {
     storeValidateAuthorization = verifiableProperties.getBoolean("store.validate.authorization", false);
     storeTtlUpdateBufferTimeSeconds =
         verifiableProperties.getIntInRange(storeTtlUpdateBufferTimeSecondsName, 60 * 60 * 24, 0, Integer.MAX_VALUE);
+    storeIOErrorCountToTriggerShutdown =
+        verifiableProperties.getIntInRange("store.io.error.count.to.trigger.shutdown", 1, 1, Integer.MAX_VALUE);
```

Comment:
you need to change this to `Integer.MAX_VALUE` :)

Reply:
My bad. Fixed.